packages/fxa-auth-server/lib/error.js

```diff
@@ -514,6 +514,15 @@ AppError.gone = function () {
   });
 };
 
+AppError.goneFourOhFour = function () {
```

Comment:
@clouserw Does this need to be a 404 or can we use a [410 Gone](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/410)? We already have this error above.

Reply:
I didn't test 410 responses, so, I'm not comfortable signing off on that.  It would probably be fine and it could be tested. :)

Reply:
My thought with 404 is that eventually we just remove all this code and it's a real 404 from nginx

Reply:
Sounds good, filed https://mozilla-hub.atlassian.net/browse/FXA-9478 to track removing it.